config/collectdata/python/write_organ_tags.py

```diff
@@ -36,29 +51,29 @@
 queryOutput = []
 for line in ret["results"]["bindings"]:
     queryOutput.append(line)
-# generate list of IRIs and organ labels
+
+# generate list of organ cell DL queries and semantic labels
 organs = []
 for n in queryOutput:
     IRI = n['x']['value'].replace("_", ":")
     IRI = IRI.partition('http://purl.obolibrary.org/obo/')[-1]
+    IRI = "cell and 'part of' some " + IRI
```

Comment:
The class expression (DL query) should use curies for all entities  - cell & part_of as well as the target organ.

Reply:
Updated var name